ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -72,6 +72,8 @@ spec:
       optional: true
     - name: registry-credentials
       optional: true
+    - name: registry-cacert
```

Comment:
Please add docs to the README in the `Operator CI Pipeline` section about the new workspace/param.

Reply:
Documentation added